server/chat-commands/moderation.js

```diff
@@ -1393,10 +1393,12 @@ exports.commands = {
 	hidetext(target, room, user, connection, cmd) {
 		if (!target) return this.parse(`/help hidetext`);
 
-		this.splitTarget(target);
+		const lineCount = parseInt(this.splitTarget(target)) || 0;
 		let targetUser = this.targetUser;
 		let name = this.targetUsername;
-		if (!targetUser && !room.log.hasUsername(target)) return this.errorReply(`User ${target} not found or has no roomlogs.`);
+		if (!targetUser && !room.log.hasUsername(name)) {
+			return this.errorReply(`User ${name} not found or has no roomlogs.`);
+		}
```

Comment:
```suggestion
		}
		if (lineCount && cmd.includes('alt')) {
			return this.errorReply(`You can't specify a line count when using /hidealtstext.`);
		}
```

Silent failure is bad! Add an error message!